apiserver/facades/client/application/application.go

```diff
@@ -2752,3 +2754,26 @@ func (api *APIBase) crossModelRelationData(rel Relation, appName string, erd *pa
 	}
 	return nil
 }
+
+func checkCAASMinVersion(ch charm.Charm, caasVersion *version.Number) (err error) {
```

Comment:
This was in the repositories.go file that got removed. It has been relocated here.